wazo_router_confd/tests/test_api_carrier_trunks_auth.py

```diff
@@ -0,0 +1,309 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from unittest import mock
```

Comment:
I'd like to see 2 tests

1. One with no `X-Auth-Token` and getting a `401`
2. One where the `Wazo-Tenant` is not accessible to the user doing the request and getting a `401`

Reply:
You have them in `test_auth_token.py`